### PR TITLE
use same defaults for naming style as ship with the dotnet sdk

### PR DESCRIPTION
**Pull request by jmarolf**

### CODE REVIEW

File: src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CompilerExtensionsResources.resx [hunk 1]
```diff
@@ -279,4 +279,67 @@
   <data name="New_line_preferences" xml:space="preserve">
     <value>New line preferences</value>
   </data>
+  <data name="camelCase" xml:space="preserve">
+    <value>camelCase</value>
```
Comment (Youssef1313) [inline src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CompilerExtensionsResources.resx:283]: This shouldn't be localized right? I'd add a "Locked" note for that.

File: src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/NamingStyles/Serialization/NamingStylePreferences.cs [hunk 2]
```diff
@@ -139,252 +140,258 @@ public override int GetHashCode()
...
-    <SerializableNamingRule SymbolSpecificationID=""2c07f5bf-bc81-4c2b-82b4-ae9b3ffd0ba4"" NamingStyleID=""87e7c501-9948-4b53-b1eb-a6cbe918feee"" EnforcementLevel=""Info"" />
-    <SerializableNamingRule SymbolSpecificationID=""5f3ddba1-279f-486c-801e-5c097c36dd85"" NamingStyleID=""87e7c501-9948-4b53-b1eb-a6cbe918feee"" EnforcementLevel=""Info"" />
-  </NamingRules>
-</NamingPreferencesInfo>
-";
+        private static readonly Lazy<string> s_defaultNamingPreferencesString = new(() => ConstructDefaultNamingPreferencesString());
+
+        private static string ConstructDefaultNamingPreferencesString()
+        {
+            using var _ = PooledStringBuilder.GetInstance(out var sb);
```
Comment (Youssef1313) [inline src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/NamingStyles/Serialization/NamingStylePreferences.cs:147]: Why not a simply a string as it was before?

Comment (jmarolf) [reply src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/NamingStyles/Serialization/NamingStylePreferences.cs:147]: found it easier to work with personally. open to putting it back

Comment (jmarolf) [reply src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/NamingStyles/Serialization/NamingStylePreferences.cs:147]: with raw string literals I would probably but it back

File: src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CompilerExtensionsResources.resx (hunk 1 above)
Comment (jmarolf) [reply src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CompilerExtensionsResources.resx:283]: good idea thanks!

**Pull request closed without merging**